src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -43,19 +40,19 @@ export interface SensitivityProps {
 export const Sensitivity: FC<SensitivityProps> = ({ value }) => {
   const classes = useStyles();
 
-  const chipClass =
-    value === 'High'
-      ? classes.chipHigh
-      : value === 'Medium'
-      ? classes.chipMedium
-      : classes.chipLow;
+  const chipClass = clsx({
+    [classes.chip]: true,
+    [classes.chipHigh]: value === 'High',
+    [classes.chipMedium]: value === 'Medium',
+    [classes.chipLow]: value === 'Low',
+  });
```

Comment:
I thought of a better way to do this, if you want to tweak it again...
```suggestion
  const chipClass = clsx(classes.chip, classes[`chip${value}`]);
```